src/containers/home/index.tsx

```diff
@@ -29,6 +34,11 @@ export const Homepage = () => {
       console.error(error);
     }
   };
+  useEffect(() => {
```

Comment:
What if the user navigates directly to a page that is not the homepage? We should still check for that so this logic should rather be performed in a wrapper

Reply:
Indeed, this type of checks could be added to a top level component like Layout